pybv/tests/test_bv_writer.py

```diff
@@ -218,32 +221,55 @@ def test_write_read_cycle(meas_date):
     rmtree(tmpdir)
 
 
-# XXX: test also binary_int16 here
 resolutions = np.logspace(0, -9, 10)
 resolutions = np.hstack((resolutions, [np.pi, 0.5, 0.27e-6, 13]))
 
 
-@pytest.mark.parametrize("format", ["binary_float32"])
+@pytest.mark.parametrize("format", SUPPORTED_FORMATS.keys())
 @pytest.mark.parametrize("resolution", resolutions)
-@pytest.mark.parametrize("unit", ["V", "mV", "uV", "µV", "nV"])
-def test_unit_resolution(format, resolution, unit):
-    """Test different combinations of units and resolutions."""
+@pytest.mark.parametrize("unit", SUPPORTED_UNITS)
+def test_format_resolution_unit(format, resolution, unit):
```

Comment:
It's really hard to see which combination of parameters are expected to fail and which are expected to pass - maybe split the test into several shorter ones?

Reply:
I agree that this test is getting out of hand in terms of readability. 

I do not explicitly know beforehand which tests will work and which won't --> the test checks in two ways:

1. data that *can* be represented is written and a roundtrip check is done
1. data that *cannot*, is checked that it raises an error

I would defer improvements of this to future PRs

Reply:
OK, can you please open an issue as a reminder to simplify this test? Ideally, you'd hardcode a couple of value combinations that work and some others that don't.

Reply:
done: https://github.com/bids-standard/pybv/issues/64